tests/test_disc_galerkin.py

```diff
@@ -0,0 +1,168 @@
+from math import isclose
+
+import pytest
+from firedrake import *
+from irksome import Dt, MeshConstant, DiscontinuousGalerkinTimeStepper
+from irksome import TimeStepper, RadauIIA
+from ufl.algorithms.ad import expand_derivatives
+import FIAT
+
+
+@pytest.mark.parametrize("order", [0, 1, 2])
+@pytest.mark.parametrize("basis_type", ["Lagrange", ])
```

Comment:
We should be testing more basis types here

Reply:
I've now tested the `basis_type` values that we've implemented.  Are there reasonable choices of `variant` that we should test as well?

Reply:
`integral` since we might be going down that road for coarsening to low-degree in time?

Reply:
`integral` is already there, for both DG and CG cases (this is a thread on an outdated commit before I added that)